guides/v2.0/config-guide/config/config-create.md

```diff
@@ -37,12 +37,12 @@ For example, to introduce an {% glossarytooltip edb42858-1ff8-41f9-80a6-edf0d86d
 
    If other modules have a `search.xml` file, they are merged with your file when it loads.
 
-To create a new configuration type, extend the `\Magento\Framework\Config\ReaderInterface`, which is [Magento\\Framework\\Config\\Reader\\Filesystem]({{ site.mage2000url }}lib/internal/Magento/Framework/Config/Reader/Filesystem.php){: target="_blank"} to provide the following parameters:
+To create a new configuration type, extend the [\Magento\Framework\Config\ReaderInterface]({{ site.mage2000url }}lib/internal/Magento/Framework/Config/ReaderInterface.php), which is [Magento\\Framework\\Config\\Reader\\Filesystem]({{ site.mage2000url }}lib/internal/Magento/Framework/Config/Reader/Filesystem.php){: target="_blank"} to provide the following parameters:
 
-*  `$fileResolver`. Implements `\Magento\Framework\Config\FileResolverInterface`. This parameter lists the files containing the configurations of your custom type.
-*  `$converter`. Implements `\Magento\Framework\Config\ConverterInterface`. This parameter converts the XML into the internal array representation of the configurations.
-*  `$schemaLocator`. Implements `\Magento\Framework\Config/SchemaLocatorInterface`. This parameter provides the full path to file(s) containing schema(s) for validation of the individual and merged configuration files.</p>
-*  `$validationState`. Implements `\Magento\Framework\Config\ValidationStateInterface`. This parameter defines whether a configuration file should be validated. 
+*  `$fileResolver`. Implements [\Magento\Framework\Config\FileResolverInterface]({{ site.mage2300url }}lib/internal/Magento/Framework/Config/FileResolverInterface.php). This parameter lists the files containing the configurations of your custom type.
```

Comment:
The varisbles in lines 42, 43, and 44 need to be changed to mage2000url

Reply:
@keharper Done.